src/platform/site-wide/sass/formation-overrides/_variables.scss

```diff
@@ -0,0 +1,30 @@
+@import "functions";
+
+$em-base:             100%; // USWDS v3 html value
+$base-font-size:      16px; // USWDS v3 body value
```

Comment:
These variables are the base font sizes that are causing us so many issues in Formation and updating the assigned variables to the USWDS values means they will cascade into other stylesheets.

I think we will still want base USWDS font sizes in the css-library and imported eventually though.